es6-promises/main.js

```diff
@@ -0,0 +1,7 @@
+import takeAChance from './take-a-chance.js';
+
+const lottery = takeAChance('Kristian');
+
+lottery.then((value) => console.log(value),
+  lottery.catch((Error) => console.log(Error.message))
+);
```

Comment:
This isn't quite right. The `.catch` should be chained directly on `.then`:
`lottery.then(...).catch(...)`